MID Server/MID Server Script Includes/IllumioAuthenticateToPCE.js

```diff
@@ -28,10 +28,17 @@ IllumioAuthenticateToPCE.prototype = {
         this.utils = new IllumioPCEUtils(this.timeZone);
         this.protocol = this.utils.getPortFromUrl(this.pceUrl);
 
+        this.retryParams = null;
+        try {
+            this.retryParams = JSON.parse(probe.getParameter('glide.jms.retry_params'));
+        } catch(e) {
+            this.logger._except('IllumioAuthenticateToPCE - Cannot parse the JSON of retry parameters');
+        }
+        
         var decodedAuth = this.utils.decodeBase64(this.pceAuthorization);
         this.pceUsername = decodedAuth.substring(0, decodedAuth.indexOf(":"));
```

Comment:
Just use `decodedAuth.split(':')` here. If the decoded password can have ':' characters, you can rejoin the result, e.g.

```
[this.pceUsername, ...password] = decodedAuth.split(':');
this.pcePassword = password.join(':');
```